### DISCUSSION TITLE
Alert Dialog on page load

### DISCUSSION THREAD

**Discussion by s1n7ax** [huntabyte/shadcn-svelte#1354] [Help] [answered]
I want to show a welcome message on page load as an alert. How to do that without the default button click trigger?

**Comment by pidanou** [comment, marked as answer]
It uses this: https://www.bits-ui.com/docs/components/alert-dialog
So you have a "open" prop

**Reply by s1n7ax** [reply]
I started using svelte for the first time. What is the difference of shadcn-svelte and bits-ui? Just the looks?

**Reply by pidanou** [reply]
`bits-ui` is a headless component library, basically it has features but no styling etc. 
https://bits-ui.com/docs/introduction 
And bits-ui is built on top of melt-ui
